app/src/main/java/nl/mpcjanssen/simpletask/util/Config.kt

```diff
@@ -257,63 +213,45 @@ object Config : SharedPreferences.OnSharedPreferenceChangeListener {
     val todoFile: File
         get() = File(todoFileName)
 
-    @SuppressLint("CommitPrefEdits")
     fun setTodoFile(todo: String) {
-        with(prefs.edit()) {
-            putString(getString(R.string.todo_file_key), todo)
-            remove(getString(R.string.file_current_version_id))
-            commit()
-        }
+        _todoFileName = todo
+        prefs.edit().remove(getString(R.string.file_current_version_id)).apply()
```

Comment:
I changed `commit()` to `apply()` here even though I didn't strictly have to (as compared to the places that use the library, which doesn't have the ability to `commit()` yet).